Dockerfile

```diff
@@ -20,31 +20,46 @@ RUN set -x \
 # Add the officially endorsed Erlang debian repository:
 # See:
 #  - http://www.erlang.org/download.html
-#  - https://www.erlang-solutions.com/downloads/download-erlang-otp
+#  - https://www.erlang-solutions.com/resources/download.html
 RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 434975BD900CCBE4F7EE1B1ED208507CA14F4FCA
 RUN echo 'deb http://packages.erlang-solutions.com/debian jessie contrib' > /etc/apt/sources.list.d/erlang.list
 
+# install Erlang
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		erlang-asn1 \
+		erlang-base-hipe \
```

Comment:
`erlang-base-hipe` is the only package not already present in line [40/-](https://github.com/aexeagmbh/docker-rabbitmq/pull/1/files#diff-3254677a7917c6c01f55212f86c57fbfL40).